Source/WebCore/dom/Element.cpp

```diff
@@ -3828,6 +3844,13 @@ AtomString Element::computeInheritedLanguage() const
     return document().contentLanguage();
 }
 
+AtomString Element::langForStyleResolution() const
+{
+    if (!hasRareData())
+        return nullAtom();
+    return elementRareData()->langForStyleResolution();
+}
+
 Locale& Element::locale() const
 {
     return document().getCachedLocale(computeInheritedLanguage());
```

Comment:
Can this use of computeInheritedLanguage() also be replaced with the new mechanism? Can all of them?

Reply:
I'm gonna do that as a follow up.